recipes/wayland/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Peter Williams and collaborators.
```

Comment:
You should remove this line, I think.

Reply:
i'll take this as your approval to remove it.